R/ForecastRegrTask.R

```diff
@@ -0,0 +1,58 @@
+#' Create a task for univariate forecasting
```

Comment:
and after @desc a linebreak

so like this 

@title blaaaaaa

@desc
sfdsd dfdsfsd
fdsfdsffdfs 

@param
@param
@return
rest